Description: Correct pod NAME issue in LedgerSMB/DBObject/Customer.pm
 Has a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472558&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Customer.pm
+++ b/LedgerSMB/DBObject/Customer.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::DBObject::Customer, LedgerSMB Class for Customers
+LedgerSMB::DBObject::Customer - LedgerSMB Class for Customers
 
 =head1 SYNOPSIS
 
